### PR TITLE
Update examples to mention stack component deploy CLI command

### PR DESCRIPTION
**Pull request by strickvl**
Updated all examples to include references to the new `zenml <STACK_COMPONENT> deploy...` subcommand.

This will need to be merged onto the main PR / implementation branch #1328.

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: examples/kserve_deployment/README.md [hunk 1]
```diff
@@ -340,6 +340,14 @@ components are described in this document, but similar stacks may be set up
 using different backends and used to run the example as long as the basic Stack
 prerequisites are met.
 
+#### Deploying Individual Stack Components using the CLI
+
+The KServe model deployer and the GCP artifact store (i.e. as individual stack
+components) can be deployed using the ZenML CLI as well, using the `zenml
```
Comment (fa9r) [inline examples/kserve_deployment/README.md:346]: "(i.e. as individual stack components)" what does this mean?

File: examples/sagemaker_orchestration/README.md [hunk 2]
```diff
@@ -93,6 +93,20 @@ zenml image-builder register local_builder --flavor=local
 zenml stack register sagemaker_stack -a aws_artifact_store -o sagemaker_orchestrator -c aws_registry -i local_builder --set
 ```
 
+Note that if you want an easy way to deploy the required cloud resources for
+this example via the ZenML CLI, you can do so using the `zenml <STACK_COMPONENT>
+deploy` sub-command. For example, to deploy the S3 artifact store, you can
```
Comment (fa9r) [inline examples/sagemaker_orchestration/README.md:98]:
```suggestion
deploy` command. For example, to deploy the S3 artifact store, you can
```

File: examples/kubernetes_orchestration/README.md [hunk 3]
```diff
@@ -67,6 +67,11 @@ to have the following additional software installed on your local machine:
 * [kubectl](https://kubernetes.io/docs/tasks/tools/)
 * [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
 
+Note that the Kubernetes orchestrator, the ECR container registry and the S3
+artifact store can be deployed using the ZenML CLI as well (i.e. as an
+alternative to the instructions below), using the `zenml <STACK_COMPONENT> deploy` command. For more information on this `deploy` subcommand, please refer to the
```
Comment (fa9r) [inline examples/kubernetes_orchestration/README.md:72]: Same here, the "(i.e. ...)" isn't needed IMO

File: examples/seldon_deployment/README.md [hunk 4]
```diff
@@ -233,6 +233,13 @@ components are described in this document, but similar stacks may be set up
 using different backends and used to run the example as long as the basic Stack
 prerequisites are met.
 
+#### Deploying Individual Stack Components using the CLI
+
+The Seldon model deployer and the AWS S3 artifact store (i.e. as individual stack
```
Comment (fa9r) [inline examples/seldon_deployment/README.md:238]: same here

File: examples/custom_code_deployment/README.md [hunk 5]
```diff
@@ -188,6 +188,15 @@ zenml artifact-store register gcs_store -f gcp \
...
 
+Note that you can deploy your GCS Store using the ZenML CLI as well, using the
+`zenml artifact-store deploy` command. This is how you would do it:
+
+```shell
+zenml artifact-store deploy gcs_store --flavor=gcs --project_id=my_project ...
+```
+
+For more information on this `deploy` subcommand, please refer to the 
+[documentation](https://docs.zenml.io/advanced-guide/practical-mlops/stack-recipes#deploying-stack-components-directly).
```
Comment (fa9r) [inline examples/custom_code_deployment/README.md:199]: In the live 0.37.0 version I don't see any documentation for the `zenml ... deploy` command on that page. Is that done on some other branch perhaps?

**Review by fa9r — CHANGES_REQUESTED** [review]
Nice, I didn't even know myself that we have this functionality already, thanks for documenting it!

The docs page you linked to doesn't seem to have any documentation on the `zenml ... deploy` command, however. Does the command itself still need to be documented there?

Apart from that it's ready to go in my opinion. Personally, I find the "(i.e. ...)" phrases a bit hard to read but I'll leave that up to you.

**Review by fa9r — APPROVED** [review]
Nevermind, just saw you documented the command in #1456, LGTM then

File: examples/kserve_deployment/README.md (hunk 1 above)
Comment (strickvl) [reply examples/kserve_deployment/README.md]: You're right. Also unnecessary to say, actually. I removed it everywhere.

**Review by wjayesh — APPROVED** [review]
Looks good, thanks for adding it to all the examples! Also, there's one extra `*` left over from previous changes on line 287.

**Pull request merged**